python/cudf/cudf/_lib/string_casting.pyx

```diff
@@ -509,7 +509,8 @@ def from_booleans(Column input_col):
 
 def int2timestamp(
         Column input_col,
-        format):
+        format,
```

Comment:
```suggestion
        string format,
```

Reply:
It actually appears that `format` is passed to this function as bytes and not as a string. Removing `string` from this line seems to have fixed the cython issues.

Reply:
What is the `bytes` like input for `format`? Could you check by putting a print statement inside this function and run the failing pytests

Reply:
I think the issue is probably that you want to use `str` and not `string`.

Reply:
I got an earlier error message that made me think format was being passed as bytes, but I think Vyas is right. `str` is working as expected here :D